.github/CODEOWNERS_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+# CODEOWNERS file to define owners for different parts of the code.
+
+# The user specified here will be responsible for all files in the repository.
+* @devFelipeAlmeida @OtherUser1 @OtherUser2
```

Comment:
Thanks @devFelipeAlmeida For now. Use my username @rt4914 as I will need to review all the code.